ledger/catchupaccessor.go

```diff
@@ -413,10 +435,14 @@ func (c *CatchpointCatchupAccessorImpl) processStagingBalances(ctx context.Conte
 	if errHashes != nil {
 		return errHashes
 	}
+	if errKVs != nil {
+		return errKVs
+	}
 
 	progress.BalancesWriteDuration += durBalances
 	progress.CreatablesWriteDuration += durCreatables
 	progress.HashesWriteDuration += durHashes
+	progress.KVWriteDuration += durKVs
```

Comment:
@jannotti Minor:  Based on usage of other durations (e.g. `HashesWriteDuration`), it looks like the primary reason to record `KVWriteDuration` is for logging in https://github.com/algorand/go-algorand/blob/e83acec3e4ee1ac265d0a99d6f9f9d5b9df6dac9/catchup/ledgerFetcher.go#L144-L154 

Any reason not to extend the log message in the PR?

Reply:
I didn't notice.  I thought I saw it going out in some other metric that I did extend.  Anyway, this deserves it too, surely.